src/frontend/context/FagsakContext.tsx

```diff
@@ -82,6 +73,40 @@ const [FagsakProvider, useFagsakRessurser] = createUseContext(() => {
             });
     };
 
+    const erBrukerUtdatert = (bruker: Ressurs<IPersonInfo>, fagsak: Ressurs<IFagsak>): boolean => {
```

Comment:
Very nice! If you also could split the two states I think this is a huge improvement :)

Reply:
Ok, done!